ironfish/src/mining/director.test.slow.ts

```diff
@@ -51,7 +52,7 @@ describe('Mining director', () => {
   let targetSpy: jest.SpyInstance
   let targetMeetsSpy: jest.SpyInstance
   let isAddBlockValidSpy: jest.SpyInstance
-  let memPool: TestMemPool
+  let mempool: TestMemPool
```

Comment:
I think if we want to do casing of the `mempool` variable this way (either is fine by me), the class should be cased `TestMempool` and `Mempool`

Reply:
reverted